reactor-test/src/main/java/reactor/test/util/TestLogger.java

```diff
@@ -216,37 +176,30 @@ public boolean isErrorEnabled() {
 
 	@Override
 	public synchronized void error(String msg) {
-		if(logCurrentThreadName){
-			this.err.format("[ERROR] (%s) %s\n", Thread.currentThread().getName(), msg);
-		} else {
-			this.err.format("[ERROR] %s\n", msg);
-		}
+		this.err.format(logContent("ERROR", msg));
 	}
 
 	@Override
 	public synchronized void error(String format, Object... arguments) {
-		if(logCurrentThreadName){
-			this.err.format("[ERROR] (%s) %s\n", Thread.currentThread().getName(), format(format, arguments));
-		} else {
-			this.err.format("[ERROR] %s\n", format(format, arguments));
-		}
+		this.err.format(logContent("ERROR", format(format, arguments)));
 	}
 
 	@Override
 	public synchronized void error(String msg, Throwable t) {
+		this.err.format(logContent("ERROR", String.format("%s - %s", msg, t)));
+		t.printStackTrace(this.err);
+	}
+
+	String logContent(String logType, String msg){
```

Comment:
can you just ensure the log levels are padded left? `[%5s]` in both format Strings should suffice

Reply:
ah I see you've fixed that directly in each method. I'll let you decide if you prefer to mutualize that padding in the `logContent` method or not. Just let me know in response to this comment.

Reply:
I noticed that WARN and INFO level logging is padded differently than the rest and I wasn't sure if there's a specific reason for that so I did it this way. There's a couple of different unit tests relying on that padding as well.

Reply:
I don't mind updating those unit tests if every log type is supposed to have that padding

Reply:
yeah it was more of a cosmetic thing, intending to align the log level part of the output. with your latest changes the unit tests are fine. it is more a question of putting the padding in the `logContent` method vs in the `info(...)` and `warn(...)` methods.

Reply:
In that case i think we can leave it how it is right now with padding in the individual methods